qemu/tests/vdi_image_create.py

```diff
@@ -0,0 +1,24 @@
+from avocado import TestError
+
+from virttest import data_dir
+from virttest.qemu_storage import QemuImg
+
+
+def run(test, params, env):
+    """
+    Convert a vdi image.
```

Comment:
This line of comments is redundant. Others LGTM.

Reply:
Updated. Thanks.